lib/solidus_feeds/feeds.rb

```diff
@@ -0,0 +1,15 @@
+module SolidusFeeds::Feeds
```

Comment:
Why using a module instead of a class, which can be more easily initialized?

Reply:
Because in this way we can both extend the base module directly (see `SolidusFeeds.extend self`) and test in isolation (see `subject(:solidus_feeds)`) in the spec.